splunk-on-ubuntu/azuredeploy.json

```diff
@@ -381,6 +420,18 @@
           "contentVersion": "1.0.0.0"
         },
         "parameters": {
+          "SplunkURL": {
```

Comment:
Good point, you must be referring to the substitution with noop `empty-resources` template. Can we add `defaultValue` to those params in that template to avoid passing them down everywhere.